projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-approval-details/order-detail-approval-details.component.html

```diff
@@ -0,0 +1,66 @@
+<ng-container *ngIf="order$ | async as order">
+  <ng-container *ngIf="order.permissionResults?.length">
+    <div class="cx-approval-header row">
+      <div class="cx-approval-label col-sm-12">
+        {{ 'orderDetails.approvalDetails.header' | cxTranslate }}
+      </div>
+    </div>
+    <div class="row">
+      <table class="table table-striped cx-approval-table">
```

Comment:
I'm not sure about the latest agreements around tables. Should all new table views use cx-table for configurablity? 
I would need to check the UI to tell if it's worth it. I guess visual consistency with existing _order details_ tables is more important. 
@tobi-or-not-tobi

Reply:
It would be nice, but we're not forcing.